west.yml

```diff
@@ -212,6 +212,15 @@ manifest:
       path: modules/crypto/tinycrypt
       groups:
         - crypto
+    - name: littlefs
+      path: modules/fs/littlefs
+      revision: 6b278f4ac18990944b7f4ecf57985dab140bd181
```

Comment:
To reference a PR in a module the syntax here would be:

`revision: pull/7/head
`